Apps/W1/ShopifyConnector/app/Integration/Codeunits/ShpfyAuthenticationMgt.Codeunit.al

```diff
@@ -0,0 +1,133 @@
+/// <summary>
+/// Codeunit Shpfy Authentication Mgt. (ID 30199).
+/// </summary>
+codeunit 30199 "Shpfy Authentication Mgt."
+{
+    Access = Internal;
+    // https://shopify.dev/api/usage/access-scopes
+    ScopeTxt: Label 'write_orders,read_all_orders,write_assigned_fulfillment_orders,read_checkouts,write_customers,read_discounts,write_fulfillments,write_inventory,read_locations,read_payment_terms,write_products,write_shipping', Locked
```

Comment:
corrupted line. expected ". Locked = true;"

Reply:
Is corrected with the commit on 12/04/2022